Sources/TuistKit/Services/InitService.swift

```diff
@@ -1,266 +1,271 @@
-                requiredTemplateOptions: requiredTemplateOptions,
-                optionalTemplateOptions: optionalTemplateOptions,
-                template: template
-            )
+        let tuistSwiftFileContent = """
+        import ProjectDescription
 
-            try await templateGenerator.generate(
-                template: template,
-                to: path,
-                attributes: parsedAttributes
-            )
+        \(tuistSwiftLine)
+        """
+        let tuistSwiftFilePath = projectDirectory.appending(component: "Tuist.swift")
+        if try await fileSystem.exists(tuistSwiftFilePath) {
+            try await fileSystem.remove(tuistSwiftFilePath)
         }
+        try await fileSystem.writeText(tuistSwiftFileContent, at: tuistSwiftFilePath)
 
-        ServiceContext.current?.alerts?
-            .success(
-                .alert(
-                    "Project generated at path \(path.pathString). Run `tuist generate` to generate the project and open it in Xcode. Use `tuist edit` to easily update the Tuist project definition."
-                )
-            )
+        try await mise(path: directory, nextSteps: &nextSteps)
 
-        ServiceContext.current?.logger?
-            .info(
-                "To learn more about tuist features, such as how to add external dependencies or how to use our ProjectDescription helpers, head to our tutorials page: https://docs.tuist.io/tutorials/tuist-tutorials"
-            )
+        ServiceContext.current?.alerts?.success(.alert("You are all set to explore the Tuist universe", nextSteps: nextSteps))
     }
 
-    // MARK: - Helpers
-
-    /// Checks if the given directory is empty, essentially that it doesn't contain any file or directory.
-    ///
-    /// - Parameter path: Directory to be checked.
-    /// - Throws: An InitServiceError.nonEmptyDirectory error when the directory is not empty.
-    private func verifyDirectoryIsEmpty(path: AbsolutePath) async throws {
-        let allowedFiles = Set(["mise.toml", ".mise.toml"])
-        let disallowedFiles = try await fileSystem.glob(directory: path, include: ["*"]).collect()
-            .filter { !allowedFiles.contains($0.basename) }
-        if !disallowedFiles.isEmpty {
-            throw InitServiceError.nonEmptyDirectory(path)
+    private func mise(path: AbsolutePath, nextSteps: inout [TerminalText]) async throws {
+        if let existingMiseTomlPath = (
+            try await [path.appending(component: ".mise.toml"), path.appending(component: "mise.toml")]
+                .concurrentFilter {
+                    try await fileSystem.exists($0)
+                }
+        ).first {
+            nextSteps.append("Add 'tuist = \'\(Constants.version!)\'' to your \(existingMiseTomlPath.basename)")
+        } else {
+            try await fileSystem.writeText("""
+            [tools]
+            tuist = "\(Constants.version!)"
+            """, at: path.appending(component: "mise.toml"))
         }
```

Comment:
Can't we run `mise use tuist@x.y.z`? mise then generate the `mise.toml` if necessary or updates the existing one.

Reply:
Yeah... I dunno why I didn't think about it.